smoke_tests_fpm.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+# TODO: we should need only one web/app.php file?
+# use $ENVIRONMENT_NAME with a default?
+ping=$(SCRIPT_FILENAME=/srv/annotations/web/app_dev.php/ping REQUEST_URI=/ping REQUEST_METHOD=GET cgi-fcgi -bind -connect 127.0.0.1:9000 | tail -n 1)
```

Comment:
most of this should go into the base image too, but it's nice to be able to smoke test `php-fpm` and the application without nginx

Reply:
👍

Reply:
Can we make the `SCRIPT_FILENAME ` dynamic to avoid knowledge of `/srv/annotations` here?

Reply:
Could introduce a `PROJECT_FOLDER` environment variable (as explicit arguments would make it difficult to use)